allennlp/nn/initializers.py

```diff
@@ -104,25 +104,26 @@ def __call__(self, module: torch.nn.Module) -> None:
         module : torch.nn.Module, required.
             The Pytorch module to apply the initializers to.
         """
+        logger.info("Initializing parameters")
         # Store which initialisers were applied to which parameters.
         not_explicitly_initialized_parameters = []
         for name, parameter in module.named_parameters():
             is_initialized = False
             for initializer_regex, initializer in self._initializers.items():
                 if re.search(initializer_regex, name):
                     initializer(parameter)
-                    logger.info("Initializing %s using %s "
```

Comment:
If you're mostly using the default initializer, anyway, I don't think this is actually that helpful, because it doesn't tell you what it was actually initialized with.  But the motivation was to reduce test output, so I didn't get a huge log to page through before actually seeing my error.  I can just change the logging for this in the tests.